modules/user_accounts/php/user_accounts.class.inc

```diff
@@ -77,9 +77,24 @@ class User_Accounts extends \DataFrameworkMenu
             'Y' => 'Yes',
             'N' => 'No',
         ];
+        // Get user
+        $factory = \NDB_Factory::singleton();
+        $user    = $factory->user();
+
+        // Without user_acounts_multisite permission, only show user sites
+        if ($user->hasPermission('user_accounts_multisite')) {
+            $sites = \Utility::getSiteList(false);
+        } else {
+            $sites = [];
```

Comment:
I believe you should be using `$user->getStudySites()` instead of reimplementing the logic

Reply:
@jeffersoncasimir getStudySites only includes study sites, but I believe in this case we want _all_ sites that the user has access to